src/lib.rs

```diff
@@ -264,8 +267,26 @@ impl LightningNode {
         })
     }
 
-    pub fn get_node_info(&self) -> NodeInfo {
-        todo!()
+    pub fn get_node_info(&self) -> Result<NodeInfo> {
+        let node_state: NodeState = self.sdk.node_info().map_err(|e| RuntimeError {
+            code: RuntimeErrorCode::NodeUnavailable,
+            msg: e.to_string(),
+        })?;
+        let rate = self.get_exchange_rate();
+
+        Ok(NodeInfo {
+            node_pubkey: node_state.id,
+            peers: node_state.connected_peers,
+            channels_info: ChannelsInfo {
+                num_channels: 0,
+                num_usable_channels: 0,
+                local_balance: (node_state.channels_balance_msat + node_state.onchain_balance_msat)
+                    .to_amount_down(&rate),
+                inbound_capacity: node_state.max_receivable_msat.to_amount_down(&rate),
+                outbound_capacity: node_state.max_payable_msat.to_amount_down(&rate),
+                total_channel_capacities: node_state.max_payable_msat.to_amount_down(&rate),
```

Comment:
This value (`total_channel_capacities`) refers to the total channel capacities. I see no way of us getting this value from the SDK's interface, but I'm also not seeing why we would need it. I propose we remove it.

Reply:
Hum... I don't quite get what that field means. It seems to be a value about the LSP, not about our current existing channels

Reply:
I'm not sure if I understand this correctly, but wouldn't `total_channel_capacities` be the sum of `inbound_capacity` and `outbound_capacity`

Reply:
Besides channel reserves, there are other amounts that are not included in the `inbound`/`output` capacity, such as commit fees, so no, unfortunately we can't get the overall channel capacity that way